src/app/inventory/store/stats.ts

```diff
@@ -334,48 +355,57 @@ function enhanceStatsWithPlugs(
     return;
   }
 
+  const existingStatsByHash: NodeJS.Dict<DimStat> = _.keyBy(existingStats, (s) => s.statHash);
```

Comment:
`NodeJS.Dict`? Maybe `_.Dictionary` instead? Though typing this is generally unnecessary.